tests/testflows/rbac/configs/clickhouse/common.xml

```diff
@@ -0,0 +1,6 @@
+<yandex>
+    <timezone>Europe/Moscow</timezone>
+    <listen_host replace="replace">0.0.0.0</listen_host>
```

Comment:
You should be ready that tests can be run on ipv6only hosts.
```
                 2ms                [clickhouse1] 2020.07.29 01:53:03.101271 [ 1 ] {} <Warning> Application: Listen [::1]:8123 failed: Poco::Exception. Code: 1000, e.code() = 99, e.displayText() = Net Exception: Cannot assign requested address: [::1]:8123 (version 20.7.1.4205). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>

```

Reply:
Updated to use `::` instead of `0.0.0.0` to allow different hosts